CHANGELOG.rst

```diff
@@ -1,10 +1,10 @@
 Changelog
```

Comment:
Renamed to CHANGELOG.rst as done in #546 and reformatted to match the changelog output generated by `towncrier`